mdx-web/src/main/java/com/mx/path/model/mdx/web/controller/AccountNumbersController.java

```diff
@@ -19,4 +19,11 @@ public final ResponseEntity<AccountNumbers> get(@PathVariable("id") String accou
     AccessorResponse<AccountNumbers> response = gateway().accounts().accountNumbers().get(accountId);
     return new ResponseEntity<>(response.getResult().wrapped(), createMultiMapForResponse(response.getHeaders()), HttpStatus.OK);
   }
+
+  @RequestMapping(value = "/users/{user_id}/accounts/{id}/account_number", method = RequestMethod.GET, produces = BaseController.MDX_MEDIA)
+  public final ResponseEntity<AccountNumbers> getMdx(@PathVariable("user_id") String userId, @PathVariable("id") String accountId) throws Exception {
+    ensureFeature("accounts");
+    AccessorResponse<AccountNumbers> response = gateway().accounts().accountNumbers().getMdx(userId, accountId);
```

Comment:
We don't need to pass the user id to the accessor. It is usually ignored in other controllers. See remote [RemoteDepositsController](https://github.com/mxenabled/path-mdx-model/blob/87eb69cf916a3e731ab77fd1d7ef27249ee247e0/mdx-web/src/main/java/com/mx/path/model/mdx/web/controller/RemoteDepositsController.java) or [AccountsController](https://github.com/mxenabled/path-mdx-model/blob/87eb69cf916a3e731ab77fd1d7ef27249ee247e0/mdx-web/src/main/java/com/mx/path/model/mdx/web/controller/AccountsController.java) for examples.

Reply:
👍